test/Microsoft.AspNet.Razor.Runtime.Precompilation.Test/project.json

```diff
@@ -23,6 +23,9 @@
     },
     "frameworks": {
         "dnx451": {
+            "frameworkAssemblies": {
+                "System.Text.Encoding": "4.0.0.0"
```

Comment:
For some reason I couldn't build the project locally - it said it needed this reference.